.cicd/test-package.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+SCRIPT_NAME='test-package.gen.sh'
+
+cat > $SCRIPT_NAME <<EOL
+#!/bin/bash
+set -eu
+
```

Comment:
ok. Should I put the the `nodeos --full-version` into a separate fold since its a basic test layer that nodeos runs after installed? We might put other stuff there eventually.

Reply:
Up to you. If you do, I would make both the installation and invocation expanded log groups.

Reply:
Emojii are [here](https://github.com/buildkite/emojis).